salt/api-gateway/config/etc-kong-kong.conf

```diff
@@ -54,57 +54,53 @@ anonymous_reports = off          # Send anonymous usage data such as error
 
```

Comment:
ah - probably. from the linked bug report above I got the impression it was a bug that had been fixed and will be released in their mythical 0.10 release. so while it doesn't work now with `/etc/kong/kong.conf`, it will in the future. shrug.